golden_test.go

```diff
@@ -32,8 +32,8 @@ func TestNewWithOptions(t *testing.T) {
 		WithIgnoreOrder(false),
 	)
 
-	if g.options.Dir != "custom" {
-		t.Errorf("Expected Dir=custom, got %s", g.options.Dir)
+	if g.options.Dir != "testdata/custom" {
```

Comment:
Some people use Windows

```suggestion
	if g.options.Dir != filepath.Join("testdata", "custom") {
```